### PR TITLE
single key audio input functionality completed

### LINKED ISSUE #4: TASK #3 single-key press & hold mechanism for listening the audio
The code is working on multiple key-single listening event which is undesirable and hard to use.
**Create a function which listen to the audio and return text based on single key press & hold mechanism**

### PR DESCRIPTION
**Pull request by Dhyanesh-Panchal**
the function is working for the single key press and hold mechanism

### CODE REVIEW

File: speechToText.py [hunk 1]
```diff
@@ -2,36 +2,57 @@
...
+    engine.runAndWait()
+
+# Function to listen to the microphone with key binding
+def listen_untill_release():
+    audio_data = bytearray()  # Store raw audio data here
 
-# Function to listen to the microphone
-def record_text():
     with sr.Microphone() as source:
-        print("Listening...")
```
Comment (ParthChavda1) [inline speechToText.py:11]: add listening statement so that user knows that computer is listening

**Review by ParthChavda1 — COMMENTED** [review]
add a exit function to terminate the code

File: speechToText.py (hunk 1 above)
Comment (Dhyanesh-Panchal) [reply speechToText.py:11]: Okay, I updated in next commit👍

**Comment by Dhyanesh-Panchal** [comment]
ability to handle exit commands is introduced in this commit

**Pull request merged**